docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Geoclimate user documentation
+
+
+
+**This documentation is under construction.**
+
+
+
+## The Geoclimate chain
+
+
+
+Geoclimate chain is a collection of spatial processes to produce vector maps of cities
```

Comment:
Copy paste the main presentation available at the root of the repo

Reply:
It was supposed to be a starting file, but no problem, I will add more informations

Reply:
thxs